### PR TITLE
Fix/settings fixes

### PR DESCRIPTION
**Pull request by MonikaTheiss**
Moved appearance into settings page. Moved profile picture and banner into edit profile page.Once again removed chips on create event. Added youtube videos . When profile picture is selected, it now immediatelly shows.

### CODE REVIEW

**Comment by pieryvdl** [comment]
Hi Monika it makes sense that this pull request is failing, because the tests is running off the ones currently on Develop I believe. So until this is merged in that test is going to fail. This is now if my logic is correct.

File: lib/view/pages/events/create_events_page.dart
Comment (pieryvdl) [inline lib/view/pages/events/create_events_page.dart:1]: I think you wanted to remove this right Monika?

Comment (MonikaTheiss) [reply lib/view/pages/events/create_events_page.dart:1]: yes, thank you for catching that one again!

**Review by pieryvdl — COMMENTED** [review]
The code looks good on my side, thank you! So glad you figured out the Youtube videos. 
Just double check the tapping of the profile picture, on my side it still chnages the banner. Rest looks all good thanks!

**Review by pieryvdl — APPROVED** [review]
Works on my side thanks Monika!!!

**Comment by shan-226** [comment]
The videos do not work on my side. I just see a red screen with the error assertion failed when I open a dropdown.

**Comment by MonikaTheiss** [comment]
> The videos do not work on my side. I just see a red screen with the error assertion failed when I open a dropdown.

this is because you ran it in web, I did not build the videos for web.

**Comment by u22520016** [comment]
I get the weird error from Piery's branch here as well.  I really don't know what to do

**Comment by pieryvdl** [comment]
> I get the weird error from Piery's branch here as well. I really don't know what to do

@u22520016 Did you change the local.properties file on Android? hopefully that will solve the issue. Otherwise I can run it through with you tomorrow.

**Review by shan-226 — APPROVED** [review]
Looks good. I found an overflow on the help page. I'll send a screenshot over the whatsapp group.

**Comment by shan-226** [comment]
It's fixed. Could you maybe make the screen scrollable. When the screen is wider it overflows at the bottom, this should be fixed by making it scrollable

**Review by shan-226 — APPROVED** [review]

**Review by shan-226 — APPROVED** [review]

**Pull request merged**